grr/server/grr_response_server/gui/ui/components/human_readable_size/human_readable_size.ts

```diff
@@ -28,7 +32,9 @@ export class HumanReadableSizeComponent {
     if (i == 0) {
       decimals = 0;
     }
+    const fixed = Math.pow(10, decimals);
 
-    return size.toFixed(decimals) + ' ' + HumanReadableSizeComponent.UNITS[i];
+    return `${(Math.floor(size * fixed) / fixed).toFixed(decimals)} ${
+        HumanReadableSizeComponent.UNITS[i]}`;
```

Comment:
Please add a comment why you choose to do this math (truncating instead of rounding) and what the math does.